### PR TITLE
Feat/add trg draft tractusx metafile

### PR DESCRIPTION
**Pull request by almadigabor**
Contains information about the `.tractusx` metafile that can be used to store data about the product/repository to process programmatically.

### CODE REVIEW

File: docs/release/trg-0/trg-0-tractusx-metafile.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: TRG 0.0 - DRAFT tractusx metafile
```
Comment (Siegfriedk) [inline docs/release/trg-0/trg-0-tractusx-metafile.md:2]: Perhaps "Repository metafile" is easier to understand?

File: docs/release/trg-0/trg-0-tractusx-metafile.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+In the root of a sub/non-leading repository create a file called `.tractusx`:
+
+```yaml
+leadingRepository: "LEADING_PRODUCT_REPOSITORY_URL"
+```
+
+## Why
+
+Having a `.tractusx` metafile helps individuals (or release management/test management/other teams) to determine basic information about a repository in the scope of __Eclipse TractusX__. The `yaml` format enables the opportunity to programatically combine repositories to a single product, determine leading product repository, read data that can be processed for different cases and even more functionality as more fields will be introduced in the future.
```
Comment (Siegfriedk) [inline docs/release/trg-0/trg-0-tractusx-metafile.md:39]: I would probably remove the hint that we might want to extend it.

Also the sentence reads slightly weird, perhaps this is better? "helps individuals, release amanagement, test management and others to dtermine...

**Review by Siegfriedk — APPROVED** [review]
good enough, please see my comments

File: docs/release/trg-0/trg-0-tractusx-metafile.md (hunk 1 above)
Comment (almadigabor) [reply docs/release/trg-0/trg-0-tractusx-metafile.md]: Agree, changes applied

File: docs/release/trg-0/trg-0-tractusx-metafile.md (hunk 2 above)
Comment (almadigabor) [reply docs/release/trg-0/trg-0-tractusx-metafile.md]: Changes applied

**Pull request merged**